meta/applications/yc_s15.md

```diff
@@ -0,0 +1,402 @@
+he is the age of our target demographic and has made himself a leader in the
+wider high school programming community. Zach runs HS Hackers, the largest
+online community for high school developers (Facebook shared a
+[post](https://www.facebookstories.com/stories/112524/building-a-community-for-high-schoolers-who-code)
+about it to 2.3m users).
+
+**What's new about what you plan to do?**
+
+* Instead of focusing on teaching a set curriculum at hack clubs, hackEDU
+  focuses on building all-inclusive families where members hack on things
+  together.
+* hackEDU chapters are led by high school students themselves, so starting a
+  new club is just as difficult as creating a movie or volunteer club
+  (generally a one-page form filed with the school and the commitment of a
+  teacher sponsor). This also abstracts us away from a direct service model
+  that's limited to scaling linearly (similar to how Airbnb doesn't own any real
+  estate, we don't have any chapter leaders on staff).
+
+**Have you incorporated, or formed any legal entity (like an LLC) yet?**
+
+Yes.
+
+**What kind of entity and in what state or country was the entity formed?**
+
+hackEDU is incorporated in the state of Delaware as a non-stock non-profit
+corporation.
+
+**If you're already incorporated, when were you? If you've had funding, how
+much and from whom?**
+
+HackEDU was incorporated in January of 2014. We've secured a small amount
+($10k) of funding from various corporate sponsors.
+
+**Have you obtained your 501(c)(3) tax-exempt status yet?**
+
+We have not, but are in the final stages of closing a fiscal sponsor and are in
+the application process.
+
+**Please provide any other relevant information about the structure or
+formation of the company.**
+
+N/A
+
+**Are any of the following true?**
+
+* You are the only founder.
+* You are a student who may return to school when the next term starts.
+* Half or more of your group can't move to the Bay Area.
+* One or more founders will keep their current jobs.
+
+* [ ] Yes
+* [x] No
+
+**If you had any other ideas you considered applying with, please list them.
+One may be something we've been waiting for.**
+
+**Please tell us something surprising or amusing that one of you has
+discovered. (The answer need not be related to your project.)**
+
+`fortune | cowsay | lolcat` speaks volumes of truth.
+
+**What convinced you to apply to Y Combinator?**
+
+Engaging with the community on Hacker News and meeting several YC founders.
+
+**How did you hear about Y Combinator?**
+
+Hacker News.
+
+### Founder Profiles
+
+#### Zach Latta
+
+**Name**
+
+Zach Latta
+
+**Email Address**
+
+zach@hackedu.us
+
+**Age**
+
+17
+
+**Gender**
+
+Male
+
+**Schools, Degrees and Years of Graduation**
+
+* El Segundo High School, 2014
+* West Los Angeles College, N/A
+
+**Employers, Position / Titles and Dates**
+
+* Yo, Lead Backend Engineer, July 2014 - September 2014
```

Comment:
Was there anything from my time at Yo that sounds impressive (like building an infrastructure to support x million users)? Might be worth working it into other parts of the application.